test/JDBC/expected/smalldatetime_date_cmp-vu-verify.out

```diff
@@ -0,0 +1,3863 @@
+GO
+~~START~~
+text
+Query Text: SELECT COUNT(*) FROM date_smalldatetime_cmp_t1 WHERE date_col <= CAST('2023-06-14' AS SMALLDATETIME)
+Aggregate
+  ->  Seq Scan on date_smalldatetime_cmp_t1
+        Filter: (date_col <= '2023-06-14 00:00:00'::smalldatetime(0) without time zone)
+~~END~~
+
+~~START~~
+text
+Babelfish T-SQL Batch Parsing Time: 0.176 ms
+~~END~~
+
+
+SELECT COUNT(*) FROM date_smalldatetime_cmp_t1 WHERE date_col <= CAST('2023-06-18' AS SMALLDATETIME) AND date_col >= CAST('2023-06-16' AS SMALLDATETIME);
+GO
+~~START~~
+text
+Query Text: SELECT COUNT(*) FROM date_smalldatetime_cmp_t1 WHERE date_col <= CAST('2023-06-18' AS SMALLDATETIME) AND date_col >= CAST('2023-06-16' AS SMALLDATETIME)
+Aggregate
+  ->  Seq Scan on date_smalldatetime_cmp_t1
+        Filter: ((date_col <= '2023-06-18 00:00:00'::smalldatetime(0) without time zone) AND (date_col >= '2023-06-16 00:00:00'::smalldatetime(0) without time zone))
+~~END~~
+
+~~START~~
+text
+Babelfish T-SQL Batch Parsing Time: 0.224 ms
+~~END~~
+
+
+SELECT COUNT(*) FROM date_smalldatetime_cmp_t1 WHERE date_col = CAST('2023-06-17' AS SMALLDATETIME);
+GO
+~~START~~
+text
+Query Text: SELECT COUNT(*) FROM date_smalldatetime_cmp_t1 WHERE date_col = CAST('2023-06-17' AS SMALLDATETIME)
+Aggregate
+  ->  Seq Scan on date_smalldatetime_cmp_t1
+        Filter: (date_col = '2023-06-17 00:00:00'::smalldatetime(0) without time zone)
+~~END~~
+
+~~START~~
+text
+Babelfish T-SQL Batch Parsing Time: 0.175 ms
+~~END~~
+
+
+SELECT COUNT(*) FROM date_smalldatetime_cmp_t1 WHERE date_col > CAST('2023-06-16' AS SMALLDATETIME);
+GO
+~~START~~
+text
+Query Text: SELECT COUNT(*) FROM date_smalldatetime_cmp_t1 WHERE date_col > CAST('2023-06-16' AS SMALLDATETIME)
+Aggregate
+  ->  Seq Scan on date_smalldatetime_cmp_t1
+        Filter: (date_col > '2023-06-16 00:00:00'::smalldatetime(0) without time zone)
+~~END~~
+
+~~START~~
+text
+Babelfish T-SQL Batch Parsing Time: 0.269 ms
+~~END~~
+
+
+SELECT COUNT(*) FROM date_smalldatetime_cmp_t1 WHERE date_col >= CAST('2023-06-16' AS SMALLDATETIME);
+GO
+~~START~~
+text
+Query Text: SELECT COUNT(*) FROM date_smalldatetime_cmp_t1 WHERE date_col >= CAST('2023-06-16' AS SMALLDATETIME)
+Aggregate
+  ->  Seq Scan on date_smalldatetime_cmp_t1
+        Filter: (date_col >= '2023-06-16 00:00:00'::smalldatetime(0) without time zone)
+~~END~~
+
+~~START~~
+text
+Babelfish T-SQL Batch Parsing Time: 0.246 ms
+~~END~~
+
+
+SELECT COUNT(*) FROM date_smalldatetime_cmp_t1 WHERE date_col BETWEEN CAST('2023-06-16' AS SMALLDATETIME) AND CAST('2023-06-18' AS SMALLDATETIME);
+GO
+~~START~~
+text
+Query Text: SELECT COUNT(*) FROM date_smalldatetime_cmp_t1 WHERE date_col BETWEEN CAST('2023-06-16' AS SMALLDATETIME) AND CAST('2023-06-18' AS SMALLDATETIME)
+Aggregate
+  ->  Seq Scan on date_smalldatetime_cmp_t1
+        Filter: ((date_col >= '2023-06-16 00:00:00'::smalldatetime(0) without time zone) AND (date_col <= '2023-06-18 00:00:00'::smalldatetime(0) without time zone))
```

Comment:
can you also construct predicate which will choose index scan?

Reply:
ignore this ^. can you please point out similar cases for smalldatetime_date_cmp_t1 tables?

Reply:
Similar [case](https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/3085/files#diff-47f3942eb8058b6cdd3ddbbb1bc645d7726a6993d7ccff1768d3ddd43c0264bdR225).